README.rst

```diff
@@ -0,0 +1,51 @@
+===================
+eventsender
+===================
```

Comment:
> When a title has both an underline and an overline, the title text may be inset, as in the first two examples above. This is merely aesthetic and not significant. Underline-only title text may not be inset. (http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html)

it's both under/overlined, so it's valid, but since it's definitely not 'aesthetic' (it's just half-edited a copy-pasta from `django-eventhandler` readme), it should be fixed, true